md2html.sh

```diff
@@ -35,13 +35,16 @@ for file in "$@"; do
   # With pandoc, convert to adoc, then to HTML
   elif [ "$md2html_tool" == pandoc ]; then
     $md2html_tool -f markdown_github "${file}" -t asciidoc -o "${file%.md}.tmp.adoc"
+    # Fix formatting issue with redundant plus signes
+    sed -r -i -e 's/`\+([^`]*)\+`/`\1`/g' "${file%.md}.tmp.adoc"
```

Comment:
do you know what is the root cause? sed tricks are very fragile, so it would be preferable to solve the issue in a different way, if possible.

Reply:
No idea what the root cause is. I only know that this is caused by pandoc. I blame the gfm converter being unstable as gfm is not as widely used as markdown. I was not able to find any info on the internet neither.